.devcontainer/Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG VARIANT="11"
```

Comment:
Would we want to fix the jdk to 11, beforehand we had the option of 8 or 11.

Reply:
Prereq is to run on 11+, not sure if app is tested with other java versions but we can add back variant to be able to change version easily

Reply:
Needs to stay 11 or higher for our tooling.